packages/@okta/vuepress-site/docs/reference/api/event-types/index.md

```diff
@@ -17,4 +17,10 @@ The following is a full listing of event types used in the [System Log API](/doc
 
 Download a CSV file with all event types: [Okta Event Types](/docs/okta-event-types.csv).
 
+>**Note:** Certain tags on the event type indicate specific behavior of the associated System Log events:<br>
+>
+> * `oie-only`: This event type is only available in Okta Identity Engine enabled orgs
```

Comment:
```suggestion
> * `oie-only`: This event type is only available in Identity Engine enabled orgs
```

Reply:
This is first mention, do we no longer require the "Okta"?